.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,68 @@
+        int: 0 if no violations, 1 if violations are found.
+    """
+    ts_ignore_found = False
+
+    for file in files:
+        try:
+            with open(file, encoding="utf-8") as f:
+                for line_num, line in enumerate(f, start=1):
+                    if re.search(TS_IGNORE_PATTERN, line.strip()):
+                        print(
+                            f"❌ Error: '@ts-ignore' found in {file} "
+                            f"at line {line_num}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)